chat-commands.js

```diff
@@ -2228,7 +2229,7 @@ exports.commands = {
 	},
 	hidetexthelp: [
 		"/hidetext [username] - Removes a locked or banned user's messages from chat (includes users banned from the room). Requires: % (global only), @ * # & ~",
-		"/hidealtstext OR /hidetextalts [username] - Removes a locked or banned user's messages, and their alternate account's messages from the chat (includes users banned from the room).  Requires: % (global only), @ * # & ~",
+		"/hidealtstext OR /hidealttext OR /hidetextalts [username] - Removes a locked or banned user's messages, and their alternate account's messages from the chat (includes users banned from the room).  Requires: % (global only), @ * # & ~",
```

Comment:
...why are there any `OR`s at all here?

We literally only use `OR` for shortcuts, not for all aliases. I'd just leave whichever one makes the most sense, which I'd argue is `/hidealtstext`.